src/main/java/world/bentobox/bentobox/blueprints/BlueprintPaster.java

```diff
@@ -273,7 +273,15 @@ private void loadChunk() {
     private void tellOwner(User user, int blocksSize, int attachedSize, int entitiesSize, int pasteSpeed) {
         // Estimated time:
         double total = (double) blocksSize + attachedSize + entitiesSize;
-        BigDecimal time = BigDecimal.valueOf(total / (pasteSpeed * 20.0D) + (chunkLoadTime / 1000.0D)).setScale(1, RoundingMode.UP);
+
+        double blocksPerCycle = 20.0D;
+        double blocksPerSecond = pasteSpeed;
+        double millisecondsToSeconds = 1000.0D;
```

Comment:
I think you may as well make these constants actual constants. i.e., final statics.